ts_scripts/lint.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Credit: https://github.com/pytorch/torchx/blob/main/scripts/lint.sh
+
+set -e
+
+if [ ! "$(black --version)" ]
+then
+    echo "Please install black."
+    exit 1
+fi
+if [ ! "$(usort --version)" ]
+then
+    echo "Please install usort."
+    exit 1
+fi
+if [ ! "$(flake8 --version)" ]
+then
+    echo "Please install flake8."
+    exit 1
+fi
+
+# cd to the project directory
+cd "$(dirname "$0")/.." || exit 1
+
+GIT_URL_1="https://github.com/pytorch/serve.git"
+GIT_URL_2="git@github.com:pytorch/serve.git"
+
+UPSTREAM_URL="$(git config remote.upstream.url)" || true
+
+if [ -z "$UPSTREAM_URL" ]
+then
+    echo "Setting upstream remote to $GIT_URL_1"
+    git remote add upstream "$GIT_URL_1"
+elif [ "$UPSTREAM_URL" != "$GIT_URL_1" ] && \
+     [ "$UPSTREAM_URL" != "$GIT_URL_2" ]
+then
+    echo "upstream remote set to $UPSTREAM_URL."
+    echo "Please delete the upstream remote or set it to $GIT_URL_1 to use this script."
+    exit 1
+fi
+
+git fetch upstream
+
+CHANGED_FILES="$(git diff --diff-filter=ACMRT --name-only upstream/master | grep '\.py$' | tr '\n' ' ')"
+
+if [ "$CHANGED_FILES" != "" ]
+then
+    # Processing files one by one since passing directly $CHANGED_FILES will
+    # treat the whole variable as a single file.
+    echo "Running linters ..."
+    for file in $CHANGED_FILES
+    do
+        echo "Checking $file"
+        usort format "$file"
+        black "$file" -q
```

Comment:
black, by default, lints at column 80. It might be helpful to increase this to 100, or preferably 120 by supplying `-l 120`, which is more in line with recent times.

Reply:
Good call, fixed